### PR TITLE
Ammar admin table update

### PR DESCRIPTION
**Pull request by ahaq0**
Added # of jobs, date created, approve a job, refactor actions

### CODE REVIEW

File: src/Components/Admin/JobPosts.js [hunk 1]
```diff
@@ -101,27 +102,59 @@ class JobPosts extends Component {
...
-    const existingJobsData = await httpGet("jobs?status=ACTIVE,UNDER_REVIEW", localStorage.getItem("idToken"));
+    const existingJobsData = await httpGet("jobs", localStorage.getItem("idToken"));
+    
+    // Go through every job and derive the # of applicants to fill in table
+    Object.keys(existingJobsData.data.jobs).forEach(function(jobID){
+      let currentJob = existingJobsData.data.jobs[jobID];
+
+      currentJob.applicantCount = currentJob.job_applications.length; 
+    })
+    console.log(existingJobsData.data.jobs);
```
Comment (ahmedhamodi) [inline src/Components/Admin/JobPosts.js:127]: Console log

File: src/Components/Admin/JobPosts.js [hunk 2]
```diff
@@ -101,27 +102,59 @@ class JobPosts extends Component {
...
+   * @param {int} jobID the ID of the job to have it's status set to rejected
+   */
   removeJob = async (jobID) => {
 
     let removedJob = {
       "job_status" : "REJECTED",
     };
     await httpPut(`jobs/${jobID}`, localStorage.getItem("idToken"), removedJob);
-    
+    this.fetchJobs();
```
Comment (ahmedhamodi) [inline src/Components/Admin/JobPosts.js:144]: Do we want to call fetch jobs after a delete? Is this to update the table?

Comment (ahaq0) [reply src/Components/Admin/JobPosts.js:144]: to update the table status

**Review by ahmedhamodi — APPROVED** [review]
lgtm

**Review by arsidada — APPROVED** [review]

**Pull request merged**